examples/internal/proto/globalconfig/grpc_annotations.yaml

```
@@ -0,0 +1,12 @@
+type: google.api.Service
+config_version: 3
+
+http:
+  rules:
+      ### First service ###
+    - selector: api.first_svc.v1.FirstService.Get
```

This should match the package definition in the protobuf file
```suggestion
    - selector: grpc.gateway.examples.internal.proto.globalconfig.first_svc.v1.FirstService.Get
```